media-gfx/pencil/pencil-0.6.4.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop qmake-utils xdg-utils
+
+DESCRIPTION="2D animation and drawing program based on Qt5"
+HOMEPAGE="https://www.pencil2d.org/"
+SRC_URI="https://github.com/pencil2d/${PN}/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${P/_/-}"
+
+src_prepare() {
+	default
+	sed -e "/^QT/s/xmlpatterns //" \
```

Comment:
I understand that this is carried from previous version but it would be nice to have a comment why it is necessary.